.github/workflows/ansible-test.yml

```diff
@@ -0,0 +1,59 @@
+# README FIRST
+# 1. replace "NAMESPACE/COLLECTION_NAME" with the correct name, ie "community/zabbix"
+# 2. If you don't have unit tests remove that section
+# 3. If your collection depends on other collections ensure they are installed, see "Install collection dependencies"
+# If you need help please ask in #ansible-devel on Freenode IRC
+
+name: CI
+on:
+# Run CI against all pushes (direct commits) and Pull Requests
+- push
+- pull_request
+
+jobs:
+
+###
+# Sanity tests (REQUIRED)
+#
+# https://docs.ansible.com/ansible/latest/dev_guide/testing_sanity.html
+
+  sanity:
+    name: Sanity (Ⓐ${{ matrix.ansible }}+py${{ matrix.python }})
+    strategy:
+      matrix:
+        ansible:
+          # It's important that Sanity is tested against all stable-X.Y branches
+          # Testing against `devel` may fail as new tests are added.
+        # - stable-2.9 # Only if your collection supports Ansible 2.9
```

Comment:
Wring indent

Reply:
I uncommented this line for testing against stable-2.9 (meaning, we're testing against Ansible stable-2.9, stable-2.10, and devel). I figured we might as well test one version older than the current version.